libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ButtonCustomParams.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.navigation.dropin
+
+import android.widget.LinearLayout
+import androidx.annotation.StyleRes
+
+data class ButtonCustomParams(
```

Comment:
Let us rename this to `MapboxExtendableButtonParams`. This can be just a regular `class`
We should also add relevant kdocs here